index.js

```diff
@@ -1,32 +1,19 @@
 require('dotenv').config();
-const _ = require('lodash');
 const moment = require('moment');
-
-const { fetchTransactions } = require('./lib/plaid');
-const {
-  updateSheet,
-  addSheet,
-  clearSheet,
-  getSheets,
-  formatHeaderRow,
-  resizeColumns,
-  duplicateSheet,
-  renameSheet
-} = require('./lib/sheets');
+const { updateSheets } = require('./providers/sheets');
+const { getTransactions } = require('./providers/plaid');
 
 (async () => {
-
-  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-  // CONSTANT DEFINITIONS
-  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-  const publicTemplateSheetId = "10fYhPJzABd8KlgAzxtiyFN-L_SebTvM8SaAK_wHk-Fw";
-
   const defaultTransactionColumns = ['date', 'amount', 'name', 'account', 'category.0', 'category.1', 'pending'];
   const defaultReferenceColumns = ['notes', 'work', 'joint'];
+  const defaultSpreadsheetProvider = 'sheets';
+  const defaultTransactionProvider = 'plaid';
+
   const transactionColumns = JSON.parse(process.env.TRANSACTION_COLUMNS || null) || defaultTransactionColumns;
   const referenceColumns = JSON.parse(process.env.REFERENCE_COLUMNS || null) || defaultReferenceColumns;
   const categoryOverrides = JSON.parse(process.env.CATEGORY_OVERRIDES || null) || [];
+  const spreadsheetProvider = JSON.parse(process.env.SPREADSHEET_PROVIDER || null) || defaultSpreadsheetProvider;
```

Comment:
Probably worth extracting this into a function `getEnvOrDefault(process.env.SPREADSHEET_PROVIDER, defaultSpreadsheetProvider)` or similar